tests/sentry/api/endpoints/test_group_autofix_setup_check.py

```diff
@@ -223,7 +200,7 @@ def test_repo_write_access_not_ready(self, mock_get_repos_and_access):
     def test_repo_write_access_no_repos(self, mock_get_repos_and_access):
         group = self.create_group()
         self.login_as(user=self.user)
-        url = f"/api/0/issues/{group.id}/autofix/setup/"
+        url = f"/api/0/issues/{group.id}/autofix/setup/?check_write_access=true"
```

Comment:
Should we also have a test where check_write_access = false?

Reply:
already tested for the empty condition at test_successful_setup which is equivalent to false